css/_content.scss

```diff
@@ -0,0 +1,62 @@
+.content {
+  width: $box-width;
+  height: 350px;
+  max-height: 350px;
+  background-color: $white;
+  overflow: auto;
+
+  .content::-webkit-scrollbar {
+    display: none;
+  }
```

Comment:
Not sassy enough, remember the use of `&:`

Reply:
done